tests/unit/processor/requester/test_requester_rule.py

```diff
@@ -0,0 +1,257 @@
+                    "requester": {},
+                },
+                TypeError,
+                "missing 2 required keyword-only arguments: 'method' and 'url'",
+            ),
+            (
+                {
+                    "filter": "message",
+                    "requester": {"method": "GET"},
+                },
+                TypeError,
+                "missing 1 required keyword-only argument: 'url'",
+            ),
+            (
+                {
+                    "filter": "message",
+                    "requester": {"method": "GET", "url": "bla"},
+                },
+                ValueError,
+                "'url' must match regex",
+            ),
+            (
+                {
+                    "filter": "message",
+                    "requester": {
+                        "method": "GET",
+                        "url": "http://the-api/endpoint",
+                        "headers": {"Authorization": "Bearer Bla"},
+                    },
+                },
+                None,
+                None,
+            ),
+            (
+                {
+                    "filter": "message",
+                    "requester": {
+                        "method": "GET",
+                        "url": "http://the-api/endpoint",
+                        "headers": {"Authorization": {}},
+                    },
+                },
+                TypeError,
+                r"must be <class \'str\'>",
+            ),
+            (
+                {
+                    "filter": "message",
+                    "requester": {
+                        "method": "GET",
+                        "url": "${field}",
+                    },
+                },
+                None,
+                None,
+            ),
+            (
+                {
+                    "filter": "message",
+                    "requester": {
+                        "method": "GET",
+                        "url": "http://the-api/endpoint",
+                        "json": '{"key": "value"}',
+                    },
+                },
+                TypeError,
+                r"must be <class \\\'dict\\\'>",
+            ),
+            (
+                {
+                    "filter": "message",
+                    "requester": {
+                        "method": "GET",
+                        "url": "http://the-api/endpoint",
+                        "params": {"key": "value"},
+                    },
+                },
+                None,
+                None,
+            ),
+            (
+                {
+                    "filter": "message",
+                    "requester": {
+                        "method": "GET",
+                        "url": "http://the-api/endpoint",
+                        "params": '{"key": "value"}',
+                    },
+                },
+                TypeError,
+                r"must be <class \\\'dict\\\'>",
+            ),
```

Comment:
I think that's a duplicate from line 94 to 105.

Reply:
done